src/components/Toast/examples.tsx

```
@@ -0,0 +1,449 @@
+
+/**
+ * Example 1: Basic Toast Notifications
+ */
+export const BasicToastExample: React.FC = () => {
+  const toast = useToast();
+
+  return (
+    <div>
+      <h2>Basic Toast Examples</h2>
+      
+      <button onClick={() => toast.showSuccess('Operation successful!')}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)